build.sbt

```diff
@@ -1,16 +1,30 @@
-addCommandAlias("ci-test", "scalafmtCheckAll; scalafmtSbtCheck; test")
-addCommandAlias("ci-docs", "github; project-docs/mdoc; headerCreateAll")
+ThisBuild / scalaVersion := "2.13.2"
+ThisBuild / organization := "com.47deg"
 
-lazy val testkit = project
-  .in(file("."))
-  .settings(moduleName := "docker-testkit-munit")
-  .settings(testkitSettings: _*)
+addCommandAlias("ci-test", "scalafmtCheckAll; scalafmtSbtCheck; mdoc; test")
+addCommandAlias("ci-docs", "github; mdoc; headerCreateAll")
+addCommandAlias("ci-publish", "github; ci-release")
 
-lazy val `project-docs` = project
-  .in(file(".docs"))
-  .aggregate(testkit)
-  .settings(moduleName := "docker-testkit-munit-project-docs")
-  .settings(mdocIn := file(".docs"))
+name := "docker-testkit-munit"
+
+testFrameworks += new TestFramework("munit.Framework")
+fork in Test := true
+
+val munit   = "org.scalameta" %% "munit"               % "[0.2.0,)" % Provided // scala-steward:off
```

Comment:
how did you figure out this version? :thinking:

Reply:
I changed the version to the first one being published for Scala `2.13` and run `ci-test` using that version 😸